requirements/requirements.txt

```diff
@@ -18,6 +18,7 @@ django-kombu==0.9.1
 django-rosetta==0.7.1
 django-tastypie==0.12.1
 djtables==0.1.1
+elasticsearch==0.4.4
```

Comment:
rawes to elastic's client I think shouldn't be too hard at least in pillowtop places. most everything i elastic for querying literally just puts in data={yourquery}, so just a few api calls and you're ok.

bulk indexing for ptop fast reindex might need to be tweaked, but since rawes kind of forced you to craft your JSON payload to send to ES exactly as you would with curl, it should be relatively straightforward to switch to the official clients. Just setup stuff like connections and the function signatures are a little different.